client/src/components/Modal/Modal.css

```diff
@@ -0,0 +1,83 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: var(--default-font);
+}
+
+.Modal {
+    width: 582px;
+    height: 309px;
+    background: var(--yellow);
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    border-radius: 3px;
+    display: flex;
+    flex-direction: column;
+}
+
+.close-modal-button {
```

Comment:
I think this needs a little more padding around it

<img width="610" alt="Screen Shot 2019-12-11 at 10 41 15 AM" src="https://user-images.githubusercontent.com/4991553/70636007-d8246e80-1c02-11ea-8a9a-28c85a555f0c.png">